apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ts

```diff
@@ -106,7 +107,7 @@ router.post(
 
 router.post(
   '/',
-  asyncHandler(async (req, res) => {
+  typedAsyncHandler<'course-instance-question'>(async (req, res) => {
```

Comment:
Why is this typed as `course-instance-question`? This route can be rendered from the course URL as well (e.g. http://localhost:3000/pl/course/1/question/1/preview). Ditto for similar routes.